semcore/input-number/src/InputNumber.jsx

```diff
@@ -267,6 +275,13 @@ class Value extends Component {
     // so, we change role to `region` here and back to `input` in handleKeyUp
     element.role = 'region';
 
+    // we could press dot second time - prevent this '1.5.'
+    if (event.key === this.separatorDecimal && value.indexOf('.') !== -1) {
```

Comment:
Maybe it should be `value.indexOf(this.separatorDecimal)`?

Reply:
oh, yes, of course... AI not so good...